extensions/wikia/MonetizationModule/styles/MonetizationModule.scss

```diff
@@ -422,3 +431,7 @@
 		padding: 10px 0 11px;
 	}
 }
+
+#monetization-amazon_video-above_title {
```

Comment:
Is there a reason why this class name doesn't match our coding conventions? https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#class-and-id-naming-conventions